src/v2/connectors/repository/ContactsRepository.js

```diff
@@ -0,0 +1,18 @@
+const queries = require('./queries/contacts');
+const Repository = require('./Repository');
```

Comment:
Can we use the Repository Class from @envage/hapi-pg-rest-api/src/repository instead? That is what we use in the water service

Reply:
I don't know. The water service queries tend to target a single table, which the hapi-rest-api demands, but the repositories in the crm don't have to be just about one table and could return data that has been joined. 

Maybe seek another opinion because I don't think the hapi-api-repo is the answer, but happy to go with the majority.